### DISCUSSION TITLE
Mystery Signature

### DISCUSSION THREAD

**Discussion by sean-mullen** [Set-OutlookSignatures/Set-OutlookSignatures#113] [General]
This script was run on a cloud only 365 tenant experimentally (details are unclear) and now every email has an extra HTML signature applied - even new accounts. 

There are no transport rules and the app registration was deleted. Can you shed any light on where this signature could potentially be stored or how it being applied? My goal is simply to remove it start over. 

Thank you

**Comment by GruberMarkus** [comment]
Hi @sean-mullen,

Do you users already see the signature when writing mails, or only after sending them?

In the first case:
- When you talk about Outlook on Windows, signatures are very likely stored in %appdata%\microsoft\signatures.
- When you talk about Outlook Web, you find them in Settings, Compose and Reply.

In the seconds case, you either use a transport rule or another tool, but not Set-OutlookSignatures.

**Reply by sean-mullen** [reply]
The extra signature appear after they are sent. 

I think it must be caused by a 365 bug because there are no app registrations or transport rules.

**Comment by GruberMarkus** [comment]
As the signatures only appear after they are sent, Set-OutlookSignatures is out of the game.

You may want to check your transport rules again, as well as your inbound and outbound connectors - you may route all your mails to an external service, which then adds the signatures. Support for this goes beyond the free and open-source version of Set-OutlookSignatures - should you be looking for help, [ExplicIT Consulting](https://explicitconsulting.at) might be the company you are looking for.

Please let me know if we can close this discussion thread as successfully answered.

**Comment by sean-mullen** [comment]
Thank you for your quick response and help. Much appreciated.
